.github/container/Dockerfile.triton

```diff
@@ -1,9 +1,10 @@
 # syntax=docker/dockerfile:1-labs
 ARG BASE_IMAGE=ghcr.io/nvidia/jax-mealkit:jax
-ARG URLREF_OXLA_TRITON=https://github.com/openxla/triton.git#llvm-head
 ARG URLREF_JAX_TRITON=https://github.com/jax-ml/jax-triton.git#main
-ARG SRC_PATH_OXLA_TRITON=/opt/openxla-triton
+ARG SRC_PATH_JAX=/opt/jax
 ARG SRC_PATH_JAX_TRITON=/opt/jax-triton
```

Comment:
Is `openxla-triton` deprecated or why is it removed completely?

Reply:
I just thought it would be clearer to label what used to be called `openxla-triton` as `triton`.
https://github.com/openxla/triton is a fork of https://github.com/openai/triton that supports using LLVM versions that are closer to HEAD.
JAX/XLA already contains a build using a tag from that fork, but it doesn't expose the Python bindings.
In these containers we build ~the same tag from the same fork, but as a CMake-driven build, including Python bindings, that spits out a wheel.